config.md

```diff
@@ -52,7 +51,7 @@ For Solaris, the mounts corresponds to fs resource in zonecfg(8).
   For the Solaris operating system, this corresponds to "dir" of the fs resource in zonecfg(8).
 * **`type`** (string, REQUIRED) The filesystem type of the filesystem to be mounted.
   Linux: *filesystemtype* argument supported by the kernel are listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
-  Windows: ntfs.
+  Windows: corresponds to the type of file system on the volume, e.g. 'ntfs'.
```

Comment:
I would be much happier if we could link to some Windows docs listing allowed types here.  “e.g. `ntfs`” doesn't help with compliance testing (which types should get checked before a runtime is deemed compliant?).